consul/src/main/scala/io/buoyant/consul/v1/AgentApi.scala

```diff
@@ -0,0 +1,31 @@
+package io.buoyant.consul.v1
+
+import com.twitter.conversions.time._
+import com.twitter.finagle.http
+import com.twitter.finagle.service.Backoff
+import com.twitter.finagle.stats.{DefaultStatsReceiver, StatsReceiver}
+import com.twitter.util.{Closable, Duration, Future}
+
+object AgentApi {
+  def apply(c: Client): AgentApi = new AgentApi(c, s"/$versionString")
```

Comment:
The same as for other APIs (I inherited it from the original implementation) - from `consul/src/main/scala/io/buoyant/consul/v1/package.scala` - it defines `package object v1` with common stuff.